spec/integration/manifest.spec.yml

```diff
@@ -12,31 +12,71 @@ spec:
       items:
         type: string
         enum:
+          - analytics_engine
+          - application_observability
+          - app_search
+          - auditd
+          - authentication
           - aws
           - azure
+          - big_data
+          - cdn_security
           - cloud
           - config_management
+          - connector
+          - connector_client
+          - connector_package
           - containers
+          - content_source
+          - crawler
+          - credential_management
           - crm
           - custom
+          - custom_logs
+          - database_security
           - datastore
+          - dns_security
+          - edr_xdr
+          - elasticsearch_sdk
           - elastic_stack
+          - email_security
+          - enterprise_serch
```

Comment:
There is a typo here. 
I see that this is also in package-registry https://github.com/elastic/package-registry/blob/main/categories/categories.yml#L30